internal/core/src/expr/ITypeExpr.h

```diff
@@ -29,6 +29,62 @@
 namespace milvus {
 namespace expr {
 
+using GenericValue = proto::plan::GenericValue;
```

Comment:
Not recommended for this method, put it in function or struct internal is better